homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -311,6 +319,9 @@
     ATTR_DRY: "dry",
     ATTR_FAHRENHEIT: "fahrenheit",
     ATTR_MOTOR_SPEED: "motor_speed",
+    ATTR_WATER_LEVEL: "water_level",
```

Comment:
Water level should be a separate sensor entity. See:
https://github.com/home-assistant/core/pull/45891#discussion_r569364120

Reply:
Removed the additional attributes.
Some other changes would also be good for this integration (in separate PR's):
- Using the humidifier entity The https://developers.home-assistant.io/docs/core/entity/humidifier/ which would allow setting the target humidity.
- Adding sensors for temperature, humidity etc to replace the current attributes